packages/front-end/components/Experiment/ResultsTable.tsx

```diff
@@ -61,13 +64,47 @@ export default function ResultsTable({
   riskVariation,
   setRiskVariation,
   statsEngine,
+  pValueCorrection,
   sequentialTestingEnabled,
 }: ResultsTableProps) {
   const domain = useDomain(variations, rows);
   const orgSettings = useOrgSettings();
   statsEngine = statsEngine ?? orgSettings.statsEngine ?? "bayesian";
   sequentialTestingEnabled = sequentialTestingEnabled ?? false;
 
+  let pValueTooltipBody = null;
+  if (sequentialTestingEnabled) {
+    pValueTooltipBody = (
+      <p className="mb-0">
+        Sequential testing is enabled. These are &apos;always valid
+        p-values&apos; and robust to peeking. They have a slightly different
+        interpretation to normal p-values and can often be 1.000. Nonetheless,
+        the interpretation remains that the result is still statistically
+        significant if it drops below your threshold (
+        {orgSettings.pValueThreshold ?? 0.05}).
+      </p>
+    );
+  }
+  if (pValueCorrection) {
+    let correctionText = "all non-guardrail metrics and variations";
+    if (tableRowAxis === "dimension") {
+      correctionText =
+        "all dimension values, non-guardrail metrics, and variations";
+    }
+    pValueTooltipBody = (
+      <>
+        {pValueTooltipBody}
+        {sequentialTestingEnabled ? <p></p> : <></>}
+        <p className="mb-0">
+          The p-values presented below are adjusted for multiple comparisons
+          using the {pValueCorrection} method. P-values were adjusted across
+          tests for {correctionText}. The unadjusted p-values are returned in
+          parentheses.
+        </p>
+      </>
```

Comment:
A couple comments

1. Any time you are adding `mb-0` to a `<p>`, you should probably just use `<div>` instead.
2. Instead of checking for `sequentialTestingEnabled` in order to render extra space, check `pValueTooltipBody` instead.  That way, it will continue working, even if we change the logic above about when we add tooltip text.
3. Instead of using `<p></p>` to add extra space, you can just wrap the first part in a div.

```tsx
<>
  {pValueTooltip && (
    <div className="mb-3">{pValueTooltip}</div>
  )}
  <div>...</div>
</>
```